nautobot/virtualization/tests/test_filters.py

```diff
@@ -49,11 +49,12 @@ def test_clusters(self):
         params = {"clusters": [self.clusters[0].pk, self.clusters[1].name]}
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
 
-        params = {"has_clusters": True}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        with self.subTest("Has Clusters"):
```

Comment:
You want to put each assertion in a new [subTest](https://docs.python.org/3/library/unittest.html#subtests) context so that if an assertion fails, the next one still runs.

Reply:
The subtests are there to make it easier for developers to troubleshoot when tests fail. If all of the assertions in a test function are in subtests then you'll see all of the failures and can fix them all without having to re-run the tests. It's not something that should prevent us from merging your changes, it's just nice to have.